common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -192,7 +190,7 @@ def test_progress_page_has_view_certificate_button(self):
 
         self.complete_course_problems()
 
-        self.course_info_page.visit()
+        self.course_home_page.visit()
```

Comment:
I'm not clear on why this is needed.  Maybe to really ensure complete_course_problems() is complete in the courseware?  It feels like it just would slow the test down.

Reply:
I'm not sure, either. I could try removing it, but I feel like that's outside the scope of this ticket and could cause more churn because of the cost of dealing with bok choy tests.

Reply:
Ok.